packages/cli/package.json

```diff
@@ -20,6 +21,8 @@
     "chalk": "4.1.0",
     "fs-extra": "9.0.1",
     "gluegun": "4.3.1",
+    "graphql": "15.3.0",
+    "graphql-tag": "2.11.0",
```

Comment:
I don't think these two are needed since they'll be brought in from the @web3api/core-js package.

Reply:
I would advise to always put dependency in package.json. I think we already have rule to enforce it (import/no-extraneous-dependencies).

From my experience with lodestar, it's quite easy to loose track of dependencies and publish code with missing dependency because code won't complain locally if this dependency is dev dependency in some other package or installed globally.

Reply:
Got it, definitely agree. Isn't this not required here though since the cli project isn't directly importing "graphql" or "graphql-tag"?

Reply:
https://github.com/Web3-API/prototype/blob/prealpha-dev/packages/cli/src/commands/query.ts#L14

Gonna remove graphql as it's not used, I think that one came with cherry picked commit